tests/ui/test_tasks.py

```
@@ -0,0 +1,337 @@
+    """Return a mock Transformation whose .json contains the given pid."""
+    t = MagicMock()
+    t.json = {"pid": pid}
+    return t
+
+
+# ---------------------------------------------------------------------------
+# agents/rero/tasks.py - rero_get_record
+# ---------------------------------------------------------------------------
+
+
+def test_rero_get_record_success(app):
```

Ruff (0.15.2)

[warning] Unused function argument: `app`

(ARG001)